Bottom: f86dd044c4a8091f130c41a1b70a7b21092a5f68
Top:    79760b9da4d5c4f5e47763a79f668e6725bdf6e9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 11:18:47 +0000

Refresh of find-c-pass-flags-to-sub-funct

---

diff --git a/src/find.c b/src/find.c
index ee8c82a..9785bbe 100644
--- a/src/find.c
+++ b/src/find.c
@@ -345,7 +345,7 @@ ADD_PV_TO_LIST (PinType *Pin, int flag)
 
 static bool
 ADD_PAD_TO_LIST (Cardinal L, PadType *Pad, int flag)
-{ 
+{
   return add_object_to_list (&PadList[L], PAD_TYPE, Pad->Element, Pad, Pad, flag);
 }
